.github/workflows/deploy.yml

```diff
@@ -147,3 +147,16 @@ jobs:
           terraform init -reconfigure -input=false -backend-config="bucket=paas-s3-broker-prod-lon-e2123d0b-d394-4594-8056-315300e7d8a8"
           terraform apply -input=false -auto-approve -var-file ../workspace-variables/staging.tfvars -var='secret_paas_app_env_values={"RAILS_MASTER_KEY":"${{secrets.RAILS_MASTER_KEY_STAGING}}"}' -var 'logstash_url=${{secrets.SYSLOG_DRAIN_URL}}'
 
+
+      - name: Deploy to production
+        if: ${{ github.event.inputs.environment == 'production' }}
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_PRODUCTION }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_PRODUCTION }}
+          TF_VAR_paas_user: ${{ secrets.GOVPAAS_PROD_USERNAME }}
+          TF_VAR_paas_password: ${{ secrets.GOVPAAS_PROD_PASSWORD }}
+        run: |
+          export TF_VAR_paas_app_docker_image=${{ env.DOCKERHUB_REPOSITORY_BASE }}${{ env.DOCKERHUB_ENV }}:${{ env.TAG }}
+          cd terraform/app
+          terraform init -reconfigure -input=false -backend-config="bucket=paas-s3-broker-prod-lon-037149fa-bf9a-4577-a066-f627d277d6c4"
```

Comment:
Nice fresh bucket, well done to remember it :D

Reply:
It's so well documented 😛